### PR TITLE
ERM-265: Add exportcontroller for JSON export

### PR DESCRIPTION
**Pull request by fereira**
This is ready for code review ERM-265

### CODE REVIEW

File: service/grails-app/controllers/org/olf/ExportController.groovy [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  def index() {
+      log.debug("ExportController::index");
+      def result = ['ExportController':'index']
+      respond(result, status:200)
+  }
+
+  /*
+   * kbart export
+   */
+  def kbart() {
```
Comment (sosguthorpe) [inline service/grails-app/controllers/org/olf/ExportController.groovy]: I think this should not be named KBART? Maybe this should be the index method above as its a none-standard export format. Then this KBART stub should be used for the upcoming export extensions.

File: service/grails-app/services/org/olf/ExportService.groovy [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+	  Title Identifiers (all known identifiers for a title including ISSN, eISSN, EZB, ZDB,...)
+	  Coverage information for a title (start date, start volume, start issue, end date, end volume, end issue) (note that a single title can have multiple coverage statements, all coverage information should be included in the JSON)
+	  URL for title in agreement
+	  Publisher
+	  E-resource type
+	  Platform
+	  Package
+	  */ 
+	  
+	  List<TitleInstance> titles = TitleInstance.createCriteria().list {
```
Comment (sosguthorpe) [inline service/grails-app/services/org/olf/ExportService.groovy]: this appears to be functionally equivalent to the [entitled named query on the TitleInstance object](https://github.com/folio-org/mod-agreements/blob/master/service/grails-app/domain/org/olf/kb/TitleInstance.groovy#L16-L27)

You can change this query to reuse that one by simply doing...
`List<TitleInstance> titles = TitleInstance.entitled.list()`

File: service/grails-app/services/org/olf/ExportService.groovy [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+// import org.olf.export.ExportConstants
+
+
+/**
+ * 
+ */
+@Transactional
+public class ExportService { 
+	
+  private List<TitleInstance> kbart() {
```
Comment (sosguthorpe) [inline service/grails-app/services/org/olf/ExportService.groovy]: Again probably shouldn't be named kbart.

**Comment by fereira** [comment]
Agree with these changes.  I added a titles.sh script for testing and left the kbart method in the controller as a placeholder.  The ExportService kbart method is now called titles.

**Pull request merged**